### ISSUE TITLE
How

### ISSUE THREAD

**Issue by SevenIsNotNumber** [kspalaiologos/malbolge-lisp#2]
How u learn malbolge?? Very cool.

**Comment by onioneffect** [comment]
I don't think you can learn it, lol. But there's algorithms out there to generate Malbolge programs and I would guess this is a super powered version of that.

**Comment by Nakilon** [comment]
I wonder why it's all in 7z.
Would be better to put these files in Github Releases instead. And in some more supported format, maybe .tar.gz

**Comment by IGLOU-EU** [comment]
> I wonder why it's all in 7z. Would be better to put these files in Github Releases instead. And in some more supported format, maybe .tar.gz

This is a Windows user 🤷
